src/main/kotlin/racingcar/PositionPrinter.kt

```diff
@@ -0,0 +1,19 @@
+package racingcar
+
+import java.io.OutputStream
+import java.io.PrintWriter
+
+class PositionPrinter(private val positions: List<Positions>, outputStream: OutputStream) {
```

Comment:
인자에 대한 기본값을 제공하기 위해 부 생성자를 여럿 만들지 말고 매개 변수의 기본값을 사용하면 어떨까요?

```suggestion
class PositionPrinter(private val positions: List<Positions>, outputStream: OutputStream = System.out) {
```

Reply:
기본값으로만 생각하고 오버로딩이란 생각을 못하고 있었던것 같습니다. 👍 
aa46459aa005d579bafe181f9e724e36639cbb6f 에서 반영하였습ㄴ디ㅏ.